pkg/spanner/client.go

```diff
@@ -143,13 +143,13 @@ func (c *Client) TruncateAllTables(ctx context.Context) error {
 		}
 	}
 
-	_, err = c.spannerClient.ReadWriteTransaction(ctx, func(ctx context.Context, txn *spanner.ReadWriteTransaction) error {
-		return txn.BufferWrite(m)
-	})
-	if err != nil {
-		return &Error{
-			Code: ErrorCodeTruncateAllTables,
-			err:  err,
+	for _, stmt := range stms {
+		_, err := c.spannerClient.PartitionedUpdate(ctx, stmt)
```

Comment:
It's better to make this invocation parallel with each other.

Reply:
fixed 5e1dea0